SWIPs/swip-honey_to_money.md

```diff
@@ -7,51 +7,66 @@ Type: Standards track
 Category: Core
 Created: 31-07-2019
```

Comment:
don't forget to update this when submitting the final version

Reply:
Thanks for pointing out!